packages/core/test/integration/query-interface.test.js

```diff
@@ -21,15 +20,12 @@ describe(Support.getTestDialectTeaser('QueryInterface'), () => {
   if (dialect.supports.schemas) {
     describe('dropAllSchema', () => {
       it('should drop all schema', async function () {
-        await this.queryInterface.dropAllSchemas({
-          skip: [this.sequelize.config.database],
+        await this.queryInterface.dropAllSchemas({ 
+          skip: [this.sequelize.config.database], 
```

Comment:
```suggestion
        await this.queryInterface.dropAllSchemas({
          skip: [this.sequelize.config.database],
```
Trying to edit from my phone but it's not going great. Maybe this helps